### PR TITLE
Add 'redistribute' attribute to cisco_ospf_vrf provider

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
This adds `redistribute` (from source protocol) support to router ospf via the `cisco_ospf_vrf` type/provider.

Note: route-maps are not optional with router ospf (redistribute from source protocol)
Note: `redistribute maximum-prefix` is not currently supported.

Travis bundler issue:  https://docs.travis-ci.com/user/languages/ruby/#bundler-20

runci test results: http://earms-trade.cisco.com/tradeui/resultsviewer.faces?ats=/auto/rtp-core/ats_shared/users/cvanheuv&client=web&host=rtp-ads-359&archive=/auto/rtp-core/ats_shared/users/cvanheuv/users/cvanheuv/archive/19-01/agents_job.2019Jan07_08:45:47.328021.zip

Resolves https://github.com/cisco/cisco-network-puppet-module/issues/531

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -506,11 +506,11 @@ Symbol | Meaning | Description
 | [cisco_ip_multicast](#type-cisco_ip_multicast)         | ✅ * | ➖ | ➖  | ➖  | ➖   | ➖  | ➖  | \*[caveats](#cisco_ip_multicast-caveats) |
 | [cisco_itd_device_group](#type-cisco_itd_device_group)           | ✅ | ➖ | ➖ | ➖ | ✅ | ➖ | ➖ |
 | [cisco_itd_device_group_node](#type-cisco_itd_device_group_node) | ✅ | ➖ | ➖ | ➖ | ✅ | ➖ | ➖ |
-| [cisco_itd_service](#type-cisco_itd_service)                     | ✅ | ➖ | ➖ | ➖ | ✅ | ➖ | ➖ | \*[caveats](#cisco_itd_service-caveats) |
+| [cisco_itd_service](#type-cisco_itd_service)                     | ✅ | ➖ | ➖ | ➖ | ✅ | ➖ | ➖ |
```
Comment (mikewiebe) [inline README.md]: Why did `cisco_itd_service` need a change?

File: README.md [hunk 2]
```diff
@@ -3532,6 +3532,12 @@ Manages a VRF for an OSPF router.
 | N9k-F    | 7.0(3)F1(1)        | 1.5.0                  |
 | N3k-F    | 7.0(3)F3(2)        | 1.8.0                  |
 
+#### <a name="cisco_ospf_vrf-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:-------------------|:-------------|
+| `redistribute`  | Minimum Module Version 1.11.0<br>No support for `redistribute maximum-prefixes` |
```
Comment (mikewiebe) [inline README.md]: Next release will be a major release so this should be `2.0.0`

**Comment by mikewiebe** [comment]
Don't forget to add an example to https://github.com/cisco/cisco-network-puppet-module/blob/develop/examples/cisco/demo_ospf.pp

File: README.md (hunk 1 above)
Comment (chrisvanheuveln) [reply README.md]: cut/paste mishap. I'll fix.

File: README.md (hunk 2 above)
Comment (chrisvanheuveln) [reply README.md]: ✔️

**Comment by chrisvanheuveln** [comment]
> Don't forget to add an example to https://github.com/cisco/cisco-network-puppet-module/blob/develop/examples/cisco/demo_ospf.pp

I already added examples to that file: https://github.com/cisco/cisco-network-puppet-module/pull/533/files#diff-47d704742bbbd4afaadba5899bd99ef8

**Pull request merged**